peps/pep-0732.rst

```diff
@@ -0,0 +1,163 @@
+PEP: 732
+Title: The Python Documentation Editorial Board
+Author: Joanna Jablonski
+Sponsor: Mariatta Wijaya
+Status: Draft
+Type: Process
+Topic: Governance
+Content-Type: text/x-rst
+Created: 14-Oct-2023
+
+
+Abstract
+========
+
+This PEP:
+
+* Establishes the Python Documentation Editorial Board
+* Proposes how the editorial board will work
+
+Motivation
+==========
+
+The Steering Council approved the creation of a 
+`Documentation Working Group <https://github.com/python/docs-community/blame/main/docs/workgroup/workgroup_charter.rst>`_
+in March 2021.
+
+The purpose of the Python documentation is to serve the 
+present and future end users of Python. As such, the core 
+development community and the greater Python documentation 
+contributors work together to achieve this:
+
+.. image:: pep-0732/docs-pep-concentric.drawio.svg
+
+
+Specification
+=============
+
+Mandate
+-------
+
+The editorial board will:
+
+* Ensure processes are in place to maintain and improve the quality of Python's documentation
+* Foster Python documentation as a community resource to serve the current and future users
+* Act in alignment with the `Python Software Foundation mission <https://www.python.org/psf/mission/#:~:text=The%20mission%20of%20the%20Python,donation%20or%20becoming%20a%20member.>`_, which is to advance the Python programming language, and to support and facilitate the growth of a diverse and international community of Python programmers
```

Comment:
Is this Chrome-specific highlighting intentional? (It doesn't work with Safari and Firefox.)

The whole page is about the PSF mission.

```suggestion
* Act in alignment with the `Python Software Foundation mission <https://www.python.org/psf/mission/>`_, which is to advance the Python programming language, and to support and facilitate the growth of a diverse and international community of Python programmers
```